src/slurmmail/cli.py

```diff
@@ -764,6 +765,12 @@ def __process_spool_file(
         msg["From"] = options.email_from_address
         msg["Date"] = email.utils.formatdate(localtime=True)
         msg["Message-ID"] = email.utils.make_msgid()
+
+        # add optional headers
+        if options.email_headers:
+            for header_name, header_value in options.email_headers.items():
+                msg[header_name] = header_value
```

Comment:
There is a danger here that a user could set a e-mail header option in `slurm.conf`, e.g. `To: bob@example.com` which will then overwrite the `To` field already set by Slurm-Mail (see later comment also).

To safe guard against this, move option headers above line 765.